### ISSUE TITLE
[NEW] Improve our VoiceOver support

### ISSUE THREAD

**Issue by rafaelks** [RocketChat/Rocket.Chat.iOS#968]
# Description

We need to make sure all the app is usable when VoiceOver is enabled.

## Progress

- [x] Connect
- [x] Auth
- [ ] Register Account
- [ ] Left side-bar (I know this is very hard to understand)
  - [ ] Button to open rooms list
  - [ ] Channel / Group / DM (type)
  - [ ] Channel / Group / DM (title)
- [ ] Messages list
  - [ ] Text
  - [ ] System messages
  - [ ] Image attachment
    - [ ] Title of the attachment
  - [ ] Video attachment
    - [ ] Title of the attachment  
  - [ ] Audio attachment
    - [ ] Title of the attachment
    - [ ] Play / Pause button
- [ ] Settings

**Comment by rafaelks** [comment]
@cardoso Shouldn't we close this one only after we have the entire app with VoiceOver support?

**Comment by cardoso** [comment]
Makes sense @rafaelks. Or we could open separate issues, but I'll keep this in the backlog for now.
